languages/cs.json

```
@@ -79,7 +79,7 @@
     "The sitekey for this hCaptcha is incorrect. Please contact the site admin if you see this.": "Sitekey pro tento hCaptcha je nesprávný. Pokud to vidíte, kontaktujte správce webu.",
     "This hCaptcha is for testing only. Please contact the site admin if you see this.": "Tento hCaptcha je určen pouze pro testování. Pokud to vidíte, obraťte se na administrátora webu.",
     "This lets you avoid future questions by registering and setting a cookie.": "To vám umožní vyhnout se budoucím otázkám registrací a nastavením souboru cookie.",
-    "To bypass our visual challenge, we offer an accessibility cookie.": "Abychom se vyhnuli naší vizuální výzvě, nabízíme soubor cookie přístupnosti.",
+    "To bypass our visual challenge, we offer an accessibility cookie.": "Abyste mohli obejít naši vizuální výzvu, nabízíme soubor cookie pro přístup. Další informace o hCaptcha Accessibility.",
```

```suggestion
    "To bypass our visual challenge, we offer an accessibility cookie.": "Abyste mohli obejít naši vizuální výzvu, nabízíme soubor cookie pro přístup.",
```

This looks incorrect. You are probably merging two keys.